tests/test_csc.py

```diff
@@ -507,9 +532,10 @@ async def test_configure_acceleration(self):
             await self.remote.cmd_configureAcceleration.set_start(
                 alimit=new_limit, timeout=STD_TIMEOUT
             )
-            data = await self.remote.evt_configuration.next(
-                flush=False, timeout=STD_TIMEOUT
-            )
+
+            await asyncio.sleep(1.0)
```

Comment:
Instead of waiting a seconds and then "getting" the event. Consider flushing the vent before sending the command and using `next` as originally designed, e.g.;

```
self.remote.evt_configuration.flush()

await self.remote.cmd_configureAcceleration.set_start(
    alimit=new_limit, timeout=STD_TIMEOUT
)

data = await self.remote.evt_configuration.next(
    flush=False, timeout=STD_TIMEOUT
)
```

Reply:
Fixed in the commit: [f03b120](https://github.com/lsst-ts/ts_mtrotator/pull/82/commits/f03b120f7da830090949ea0b1a365e87fd3689b1). Thanks!